src/main/java/org/cactoos/text/IsEmpty.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2020 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.text;
+
+import org.cactoos.Scalar;
+import org.cactoos.Text;
+
+/**
+ * Determines if text is empty or no.
+ * @since 0.47
+ */
+public final class IsEmpty implements Scalar<Boolean> {
+
+    /**
+     * The text.
+     */
+    private final Text origin;
```

Comment:
@kokodyn `origin` is a good name for a decorated object. Here it's not the case

Reply:
@andreoss 
I see in most classes implementing Scalar we use "orgin". For example 
https://github.com/yegor256/cactoos/blob/master/src/main/java/org/cactoos/text/IsBlank.java
I have no idea for a better name
Could you propose better name, please?

Reply:
@kokodyn I would use `txt` or `text`. `origin` seems strange for a regular dependency.

Reply:
@andreoss fixed